matlab/src/cpp/arrow/matlab/tabular/proxy/record_batch.cc

```diff
@@ -18,17 +18,42 @@
 #include "libmexclass/proxy/ProxyManager.h"
 
 #include "arrow/matlab/array/proxy/array.h"
+#include "arrow/matlab/array/proxy/wrap.h"
+
 #include "arrow/matlab/error/error.h"
 #include "arrow/matlab/tabular/proxy/record_batch.h"
 #include "arrow/type.h"
 #include "arrow/util/utf8.h"
 
+#include "libmexclass/proxy/ProxyManager.h"
+#include "libmexclass/error/Error.h"
+
+#include <sstream>
+
 namespace arrow::matlab::tabular::proxy {
 
+    namespace {
+        libmexclass::error::Error make_empty_record_batch_error() {
+            const std::string error_msg =  "Numeric indexing using the column method is not supported for record batches with no columns.";
+            return libmexclass::error::Error{error::RECORD_BATCH_NUMERIC_INDEX_WITH_EMPTY_RECORD_BATCH, error_msg};
+        }
+
+        libmexclass::error::Error make_invalid_numeric_index_error(const int32_t matlab_index, const int32_t num_columns) {
+            std::stringstream error_message_stream;
+            error_message_stream << "Invalid column index: ";
+            error_message_stream << matlab_index;
+            error_message_stream << ". Column index must be between 1 and the number of fields (";
```

Comment:
"number of fields" -> "number of columns"

Reply:
thanks for noticing this.